### PR TITLE
Implemented random data for seeding.

### PR DESCRIPTION
**Pull request by jonyang221**
Implemented random data for seeding.

### CODE REVIEW

File: database/db.js [hunk 1]
```diff
@@ -2,16 +2,60 @@ var mongoose = require('mongoose');
...
 })
 
 // create model
-const AReview = mongoose.model('Review', AReviewSchema);
+var AReview = mongoose.model('review', AReviewSchema);
 
+// random name generator
+var RandomNameGenerator = () => {
+  let firstName = ["Julia", "Rob", "Jenny", "Kai", "Jon"],
+    lastName = ["Gens", "Nolan", "Shamoo", "Dong", "Yang"]
```
Comment (j-gens) [inline database/db.js:18]: lol!

File: database/db.js [hunk 2]
```diff
@@ -2,16 +2,60 @@ var mongoose = require('mongoose');
...
+      reviewerName: RandomNameGenerator()
+    })
+    newReview.save();
+    console.log(newReview)
+  })
+});
+
+// database find
+const getReviews = (callback) => {
+  AReview.find({}, (err, results) => {
```
Comment (j-gens) [inline database/db.js:49]: Question here - are you always going to grab all the reviews from the database?  

I might have missed it somewhere... but will you be selecting a random number of reviews for each movie (ex: one movie has 15 reviews and one has 27 reviews)?  Or do you plan on implementing that on the client end (if that makes sense)? :)

**Review by j-gens — COMMENTED** [review]
You've done a lot of work!  I like your solution to rendering the stars for each review - very nifty!

File: database/db.js (hunk 2 above)
Comment (jonyang221) [reply database/db.js:49]: Haven't decided yet but it makes more sense to only grab the reviews as needed.  In that case, I'll need to assign movie titles to each review and revise the getReviews function to include movie name.

Thanks for catching that!

File: database/db.js (hunk 1 above)
Comment (Evillivekai) [reply database/db.js:18]: Nice random generator, I got some similar idea as yours. BTW, the name options lol...

File: server/server.js [hunk 3]
```diff
@@ -1,12 +1,21 @@
...
-app.use(bodyParser.urlencoded({extended: true}));
+const db = require('../database/db.js');
 
+app.use(express.urlencoded({ extended: true }));
+app.use(express.json());
 
+app.get('/api/audienceReviews', (req,res) => {
+  db.getReviews((err, results) => {
+    if (err) {
+      console.log('error occurred', err);
```
Comment (therobinkim) [inline server/server.js:13]: Always end the response

File: client/src/Components/FakeAudReviews.jsx [hunk 4]
```diff
@@ -1,33 +1,46 @@
...
+          stars.push(<span className="fa fa-star checked"></span>)
+      } else {
+          stars.push(<span className="fa fa-star"></span>)
+      }
+    }
+    return stars;
+  }
+
+  return (
   <div className="recentReviews">
```
Comment (therobinkim) [inline client/src/Components/FakeAudReviews.jsx:20]: Use CSS modules or styled components

**Pull request merged**